Data/Aeson/Types/FromJSON.hs

```diff
@@ -240,6 +240,15 @@ class GFromJSON arity f where
     -- or 'liftParseJSON' (if the @arity@ is 'One').
     gParseJSON :: Options -> FromArgs arity a -> Value -> Parser (f a)
 
+    -- | An internal method that return an IResult, which is easier to
```

Comment:
I suppose this is not a breaking change since the method is not exported and it has a default?